model/src/main/proto/format_import_proto_library.bzl

```diff
@@ -0,0 +1,39 @@
+
+def format_import_proto_library(name, src, deps):
+  """
+  Formats the import statement of the src file to contain a full path to the file
```

Comment:
Nit: please make sure sentences end with punctuation in doc strings.

Also, it's worth mentioning _why_ this macro exists to provide context on when callers may want to use it.

Reply:
Okay! Done!